App/app/src/main/java/com/wokabel/app/wokabel/views/Welcome.java

```diff
@@ -7,15 +7,47 @@
 import android.widget.EditText;
 
 import com.wokabel.app.wokabel.R;
+import com.wokabel.app.wokabel.models.Vocable;
+import com.wokabel.app.wokabel.services.vocabularyTest.VocabularyTest;
 import com.wokabel.app.wokabel.WokabelApplication;
 import com.wokabel.app.wokabel.services.preferences.Settings;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class Welcome extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_welcome);
+
+        ArrayList<Vocable> testVocables;
+        // HACK: Temp test
+        testVocables = new ArrayList<Vocable>(Arrays.asList(
+                new Vocable("tree", new ArrayList<String>(Arrays.asList("Baum")), "Fängt mit B an und existiert im Wald.", "0"),
+                new Vocable("car", new ArrayList<String>(Arrays.asList("Auto")), "Brum brum", "0"),
+                new Vocable("flower", new ArrayList<String>(Arrays.asList("Blume")), "Gibts auf Wiese und im Laden", "0"),
+                new Vocable("to pay", new ArrayList<String>(Arrays.asList("bezahlen")), "Nach dem Kauf ....... man", "0"),
+                new Vocable("house", new ArrayList<String>(Arrays.asList("Haus")), "Da wohnst du drinnen", "0"),
+                new Vocable("cellphone", new ArrayList<String>(Arrays.asList("Handy")), "Nutzt du jeden Tag", "0"),
+                new Vocable("table", new ArrayList<String>(Arrays.asList("Tisch")), "Kann man was draufstellen", "0"),
+                new Vocable("bike", new ArrayList<String>(Arrays.asList("Fahrrad")), "2 Räder, selber treten", "0"),
+                new Vocable("to jump", new ArrayList<String>(Arrays.asList("springen")), "Gegeneil von fallen", "0"),
+                new Vocable("to fall", new ArrayList<String>(Arrays.asList("fallen")), "Gegenteil von springen", "0"),
+                new Vocable("forest", new ArrayList<String>(Arrays.asList("Wald")), "Den ...... vor lauter Bäumen nicht mehr sehen", "0")
+        ));
+        VocabularyTest requestAlgorithm = new VocabularyTest(testVocables, VocabularyTest.Modes.RANDOM);
+        try {
+            System.out.println(requestAlgorithm.getQuestion());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
macht es nicht eher sinn das in einem File zu saven? @atalantus

Reply:
Was sollte man bei dem Code Ausschnitt in einem File saven?